tool/tyber/go/parser/header.go

```diff
@@ -29,14 +31,19 @@ type Header struct {
 		Node struct {
 			Preset   string `json:"preset"`
 			Protocol struct {
-				SwarmListeners        string        `json:"swarmListeners,omitempty"`
-				IPFSAPIListeners      string        `json:"iPFSAPIListeners,omitempty"`
-				IPFSWebUIListener     string        `json:"iPFSWebUIListener,omitempty"`
-				Announce              string        `json:"announce,omitempty"`
-				NoAnnounce            string        `json:"noAnnounce,omitempty"`
-				LocalDiscovery        bool          `json:"localDiscovery,omitempty"`
-				Ble                   bool          `json:"ble,omitempty"`
-				MultipeerConnectivity bool          `json:"multipeerConnectivity,omitempty"`
+				SwarmListeners    string `json:"swarmListeners,omitempty"`
```

Comment:
`swarmListeners` -> starting with lowercase
`Format` -> starting with uppercase

this inconsistency is very strange

Reply:
I did not really touch this part, this is how tyber is initialized but most of the fields in this struct are not used, I will refactor it in another PR

Reply:
No need @n0izn0iz, I think the best thing to do is to wait until the `initutil` package is moved out of internal so we can remove this duplicated structure.